src/commands/user-commands/tutor-log.ts

```diff
@@ -0,0 +1,103 @@
+import { Command } from "../../interfaces/Command";
+import { commandLog, sendError } from "../../utils/logs";
+import { SlashCommandBuilder } from "discord.js";
+import { reportOptions } from "../../utils/options";
+import { TutorLogInsert } from "../../utils/types";
+import { getContactId, getTutorId, insertTutorLog } from "../../utils/supabase";
+import { createEmbeded } from "../../utils/embeded";
+
+
+export const tutorlog: Command = {
+  
+    data : new  SlashCommandBuilder()
+      .setName("tutor-log")
+      .setDescription("Log your tutor hours")
+      .addStringOption((option) =>
+      option
+      .setName("tutoring-type")
+      .setDescription("Online or in-person?")
+      .addChoices(
+        {name: 'In Person', value: 't-ip'},
+        {name: 'Online Voice Chat', value: 't-ov'},
+        {name: 'Online Text Chat', value: 't-ot'}
+        )
+      .setRequired(true)
+  )
+  .addStringOption((option) =>
+  option
+      .setName("tutored-user")
+      .setDescription("Name of the student you tutored")
+      .setRequired(true)
+  )
+    .addNumberOption((option) =>
+      option
+        .setName("hours")
+        .setDescription("How many hours did you tutor?")
+        .setRequired(true)
+    )
+    .addStringOption((option) =>
+    option
+    .setName("description")
+    .setDescription("Optional description of the tutoring session")
+    .setRequired(false)
+    )
+  ,
+    run: async (interaction) => {
+        await interaction.deferReply({ ephemeral: false });
```

Comment:
For now I'd say to keep this command reply ephemeral so that people aren't spamming up any chats with their tutor logs. If we need verification that someone logged their hours we can just check the log channel.